packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -120,6 +122,7 @@ def __init__(self,
         token = token if token else odahuflow.sdk.config.API_TOKEN
         issuer_url = issuer_url if issuer_url else odahuflow.sdk.config.ISSUER_URL
 
+        self._url = calculate_url(base_url, token, model_route, model_deployment, url_prefix)
```

Comment:
If I get it right, this will still fail if user passes `client_id`, `client_secret` and `issuer_url` instead of token. Please resolve it.